src/common/transformations/tests/symbolic_transformations/nop_broadcast.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/symbolic_transformations/nop_broadcast.hpp"
+
+#include <gtest/gtest.h>
+
+#include <openvino/core/model.hpp>
+#include <openvino/op/broadcast.hpp>
+#include <openvino/op/maximum.hpp>
+#include <openvino/op/parameter.hpp>
+#include <openvino/op/relu.hpp>
+#include <openvino/op/shape_of.hpp>
+
+#include "common_test_utils/ov_test_utils.hpp"
+#include "openvino/core/dimension_tracker.hpp"
+
+using namespace ov;
+using namespace ov::op;
+using namespace std;
+
+namespace {
+void label_shape(ov::PartialShape& shape, size_t start_label = 42) {
+    auto table = std::make_shared<ov::TableOfEquivalence>(42);
```

Comment:
Should `42` be replaced with `start_label`?

Reply:
I don't use it anymore, so I deleted the parameter. Thank you!